vm/actor/src/builtin/power/mod.rs

```diff
@@ -95,10 +96,10 @@ impl Actor {
                 Serialized::serialize(MinerConstructorParams {
                     owner: params.owner,
                     worker: params.worker,
+                    control_addresses: Default::default(),
                     seal_proof_type: params.seal_proof_type,
-                    peer: params.peer,
-                    multiaddrs: params.multiaddrs,
-                    control_addrs: Default::default(),
+                    peer_id: params.peer.0,
+                    multi_address: params.multiaddrs,
```

Comment:
This name should be changed (it's plural addresses). The type is still incorrectly just one slice of bytes